core/src/test/java/com/datastax/oss/driver/internal/core/type/codec/MapCodecTest.java

```diff
@@ -118,6 +118,16 @@ public void should_decode_non_empty_map() {
         .containsEntry("c", 3);
   }
 
+  @Test
+  public void should_decode_map_with_null_elements() {
+    when(keyCodec.decode(Bytes.fromHexString("0x10"), ProtocolVersion.DEFAULT)).thenReturn("a");
+    when(valueCodec.decode(Bytes.fromHexString("0x0002"), ProtocolVersion.DEFAULT)).thenReturn(2);
+    assertThat(decode("0x" + "00000002" + "0000000110" + "FFFFFFFF" + "FFFFFFFF" + "000000020002"))
+        .containsOnlyKeys("a", null)
+        .containsEntry("a", null)
+        .containsEntry(null, 2);
```

Comment:
I think that we are missing a test case for key and value null for the same entry but I don't know if such a test gives much value.

Reply:
It wouldn't increase coverage so I would say no.